components/Op/OpTabs.js

```diff
@@ -46,9 +46,9 @@ const opEditTab =
 
 const isNotProd = process.env.NODE_ENV !== 'production'
 
-export const OpTabs = ({ op, onChange, canManage, defaultTab }) => (
+export const OpTabs = ({ op, onChange, canManage, defaultTab, author }) => (
   <>
-    <VTabs size='large' defaultActiveKey={defaultTab} onChange={onChange}>
+    <Tabs style={shadowStyle} size='large' defaultActiveKey={defaultTab} onChange={onChange}>
```

Comment:
I'm unclear why you are changing the tabs. not required for your story.

Reply:
That happened due to a merge conflict, my branch was outdated initially. I've changed it back to what it should be.